addons/website_slides/static/src/xml/website_slides_unsubscribe.xml

```diff
@@ -0,0 +1,27 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<templates xml:space="preserve">
+
+    <t t-name="website.slides.unsubscribe.modal">
+        <div>
+            <div class="o_w_slide_unsubscribe_modal_container">
+                <t t-call="website.slides.unsubscribe.modal.subscription"/>
+            </div>
+        </div>
+    </t>
+
+    <t t-name="website.slides.unsubscribe.modal.subscription">
+        <form class="clearfix">
+            <div class="form-group row">
+                <div class="controls col-md-1 mt8">
+                    <input id="subscribed" name="subscribed" type="checkbox"/>
+                </div>
+                <label for="subscribed" class="col-form-label" style="font-weight: normal">Subscribe to get notified when a new content is added</label>
```

Comment:
Don't use inline style.  (never never)
use BS4 classes instead if it's possible, custom css class otherwise.(here you can use `font-weight-normal` bs4 class)